coding-concepts.js

```diff
@@ -67,7 +77,11 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
+
+// a) Your answer: George, Alpha, 2022
```

Comment:
Your research is great. The answer will appear as an object since the variable is creating a new instance of that Learn class. Classes are the building blocks of objects.